source/includes/_endpoint-batch.md

```diff
@@ -39,20 +41,62 @@ Must contain the URL of a dataset that the current user can read. This action wi
 }
 ```
 
-If you wish to include only certain variables from the source dataset you're appending from, you can include an ``function`` in the body of the entity.  The function should be a ZCL function name.  Its arguments should be present in the ``args`` key in the body.
+Note that by default only public and not hidden variables from the dataset will
```

Comment:
So we should include discarded variables then? I followed that rule from the export that by default does not include hidden variables since they're expected to be protected by users. 

When the user is editor on both datasets, the behavior remains. Only when the user is a viewer on the bottom dataset is that they won't have access to the hidden variables which I think is what YG expects.